src/api/app/controllers/webui/request_controller.rb

```diff
@@ -28,7 +28,7 @@ def show
                                           diffs: true, action_id: action_id.to_i, cacheonly: 1).first
       @active_action = @bs_request.bs_request_actions.find(action_id)
 
-      @request_reviews = @bs_request.reviews.for_non_staging_projects
+      @request_reviews = @bs_request.reviews.for_non_staging_projects(Project.find_by_name(@bs_request.target_project_name))
```

Comment:
This is now repeating the same database query which happens a few lines below on line 45. Please, avoid this by storing the result in a variable.